dvc/remote/ssh.py

```diff
@@ -29,6 +29,7 @@ def _exists(chunk_and_channel):
 
         with self.tree.ssh(path_infos[0]) as ssh:
             channels = ssh.open_max_sftp_channels()
+            # NOTE: is the config-based max workers limit relevant here?
```

Comment:
Hi, @efiop. May I ask for some clarification of the design?

 I'm not sure, how and should I use the configuration-based jobs (workers) limit here? Why is it based on sftp channels count there and on the CPU cores count in the base tree?